### PR TITLE
STIJ-205: Added deploy script

### PR DESCRIPTION
**Pull request by meirege**
## Description
Deploy script to be used as `./deploy.sh --type=patch`.

I only have a very basic knowledge of bash script. Input couldn't hurt @daften :)

## Motivation and Context
Make deploy easier for admins. It uses the gulp bump command. A parameter of --type=xxx is required to update to the correct version.
Right now it does the following:
- git checkout develop
- git pull latest development branch
- gulp build --> ensure latest build version of style guide
- gulp bump (with flag after gulp deploy command)
- git add .
- git commit -m 'Updated to version ' + version_number
- git tag version_number
- git push && git push --tags 

The version number for git tagging and the commit message is read dynamically from the package.json file after running gulp bump.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the CHANGELOG accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  | sed 's/[",]//g' \
+  | tr -d '[[:space:]]')
+
+echo $PACKAGE_VERSION
+
+# Add everything to git.
+echo "Adding changes to git and tagging "$PACKAGE_VERSION
+git add .
+git commit -m "Updated to version "$PACKAGE_VERSION
+git tag $PACKAGE_VERSION
```
Comment (Jeroen005) [inline deploy.sh]: git push -> merge to master -> git tag -> git push --tags

Comment (daften) [reply deploy.sh]: Agreed, needs be changed to merge to master first, then tag, then push.
Maybe the name should be different too, this does not deploy, it tags a version, so maybe version_tag.sh?
If ever needed, we can split this in version_prepare.sh en version_tag.sh

Comment (meirege) [reply deploy.sh]: I adapted the code. It now:
1. pulls the latest changes from origin/develop
2. bumps the version
3. creates a new commit
4. checks out master
5. pulls the latest changes from origin/master
6. merges develop in master
7. tags based on the package.json VERSION
8. and pushes master, develop and the git tags to the remote.

Comment (meirege) [reply deploy.sh]: I also changes the .sh name to "version_tag.sh"

**Pull request merged**